packages/syft/src/syft/service/enclave/enclave.py

```diff
@@ -45,20 +45,25 @@ class EnclaveInstance(SyftObject):
     __repr_attrs__ = ["name", "route", "status", "metadata"]
     __attr_unique__ = ["name"]
 
-    # TODO replace the create method with pydantic field validators, or find a better alternative
+    @model_validator(mode="before")
     @classmethod
-    def create(cls, route: NodeRouteType) -> Self:
-        # TODO: find the standard method to convert route to client object
-        metadata = login_as_guest(url=route.host_or_ip, port=route.port).metadata
-        if not metadata:
-            raise SyftException("Failed to fetch metadata from the node")
-        return cls(
-            node_uid=UID(metadata.id),
-            name=metadata.name,
-            route=route,
-            status=cls.get_status(),
-            metadata=metadata,
-        )
+    def initialize_values(cls, values: dict[str, Any]) -> dict[str, Any]:
+        if "route" in values:
+            route = values["route"]
+            metadata = login_as_guest(url=route.host_or_ip, port=route.port).metadata
+            if not metadata:
+                raise SyftException("Failed to fetch metadata from the node")
+
+            values.update(
+                {
+                    # 'id': UID(metadata.id),
```

Comment:
`id` is the identifier for this object's instance in the owner domain, while `node_uid` identifies the Enclave node that this object points to.

Reply:
Since this comment implies the wrong thing, let's remove it to avoid any confusion.